Podfile.lock

```diff
@@ -46,7 +46,7 @@ PODS:
   - KarhooSDK (1.3.0):
```

Comment:
Perhaps we need to wait for KarhooSDK 1.4.0 to be published so we can use it here

Reply:
Its already using the latest KarhooSDK by default